app/src/pages/review.tsx

```diff
@@ -1,180 +1,151 @@
-import { DefaultState, useAppContext } from '@context/state'
```

Comment:
Notes for reviewers: The Review page (`/review`) is the most complicated page because it handles submitting the form that's been being created throughout the form wizard.

The back link on this page routes to `/contact` and the call to action button routes the user to `/confirmation`. There are so many data validation guards on this page. Every form wizard page must be filled out with valid data in order for the user to access this page.

![Screen Shot 2022-10-28 at 2 57 36 AM](https://user-images.githubusercontent.com/67701/198563077-33cd31b3-7bdd-4596-8a13-3e1d43344537.png)